assets/css/cocktailLocate.css

```diff
@@ -1,64 +0,0 @@
-.background-image {
-    background-image: url(../images/cocktails.jpg);
-    filter: blur(5px);
-    padding: 0;
-    height: 100%;
-    background-size: cover;
-    position: absolute;
-    width: 100%;
-    background-position: center bottom;
-   
-}
-
-.container {
-    padding-top: 30px;
-    text-align: center;
-    justify-content: center;
-   
-
-}
-
-.hero {
-    position: relative;
-}
-
-.hero.is-medium .hero-body {
-    padding: 0;
-    height: 300px;
-}
-
-.title {
-    font-family: "Cormorant", sans-serif;
-    font-size: 80px;
-    filter: blur(0);
```

Comment:
I added "margin-top: 24px;" to center it more in the here; to the .title css on my branch of this.  we can see how the merge changes it here.